Index: cmt/Makefile.RootCore
===================================================================
--- cmt/Makefile.RootCore	(revision 688661)
+++ cmt/Makefile.RootCore	(working copy)
@@ -6,16 +6,16 @@
 # https://twiki.cern.ch/twiki/bin/viewauth/Atlas/RootCore#Package_Makefile
 
 PACKAGE          = JetRec
-PACKAGE_PRELOAD  = 
-PACKAGE_CXXFLAGS = -DNO_JET_VARIABLER
-PACKAGE_OBJFLAGS = 
-PACKAGE_LDFLAGS  = 
-PACKAGE_BINFLAGS = 
-PACKAGE_LIBFLAGS = 
+PACKAGE_PRELOAD  =
+PACKAGE_CXXFLAGS =
+PACKAGE_OBJFLAGS =
+PACKAGE_LDFLAGS  = -lVariableR
+PACKAGE_BINFLAGS =
+PACKAGE_LIBFLAGS =
 PACKAGE_DEP      = Asg_FastJet xAODCaloEvent xAODJet xAODMuon xAODEventInfo JetEDM JetInterface EventShapeInterface
-PACKAGE_TRYDEP   = 
-PACKAGE_CLEAN    = 
-PACKAGE_NOGRID   = 
+PACKAGE_TRYDEP   =
+PACKAGE_CLEAN    =
+PACKAGE_NOGRID   =
 PACKAGE_PEDANTIC = 0
 PACKAGE_NOOPT    = 0
 PACKAGE_NOCC     = 0
